movie-search.js

```diff
@@ -4,15 +4,29 @@ const cheerio = require('cheerio')
 const search = encodeURI(process.argv[2])
 const URL = 'http://www.imdb.com/find?ref_=nv_sr_fn&q=' + search + '&s=all'
 
+const printTitles = (rawData) => {
+  const $ = cheerio.load(rawData)
+  const titles = []
+  //There are multiple sections called .findSection, we only need the movies
+  //so we look for the first .findSection, and grab the .result_text from it
+  $('.findSection')
+    .first()
+    .find('.result_text')
+    .each((i, elem) => {
```

Comment:
I used `map` here, so I am curious as to the difference, I did read the docs, but I still would like to compare the output with you sometime.